src/main.rs

```diff
@@ -124,7 +124,7 @@ impl EventHandler for Handler {
 
     fn ready(&self, ctx: Context, ready: Ready) {
         info!("Connected as {}", ready.user.name);
-        ctx.set_activity(Activity::listening("distressed sobbing noises"))
+        ctx.set_activity(Activity::playing("Jailor"))
```

Comment:
change to Listening to "prisoners' cries"

Reply:
done